requirements.txt

```diff
@@ -1,10 +1,11 @@
-connexion
-gdal
-geoalchemy2
-geojson
-iso8601
-numpy
-psycopg2
-sentinel5dl>=0.4
-sqlalchemy
-swagger-ui-bundle
+gdal>=2.2.0
+numpy>=1.10.0
```

Comment:
the order changed. is that a problem if i install those requirements in this given order? wasn't there a thing that the gdal python bindings need numpy installed?

Reply:
The order does not matter and you cannot rely on any particular installation order. This just orders them alphabetically, except for the C-based libs, since that makes it easier to read and maintain.